styles/components/_listingCard.scss

```diff
@@ -292,6 +287,14 @@
       display: none;
     }
 
+    .additionalOverlay,
+    .editOverlay {
+      height: 100%;
+      box-sizing: border-box;
+      align-items: center;
+      padding-left: 13px; // position the first button in the middle of the thumbnail
```

Comment:
Is this padding super necessary? When I toggle it, it only adjusts things by a few pixels and I'm not noticing a difference?